SCF/SCF-3/Calculator.java

```diff
@@ -0,0 +1,153 @@
+
+public class Calculator implements GeneralCalculator {
+	

```

Comment:
Need one more method to check whether a number is a valid Hexadecimal no or not. (Low Priority)